lib/client-assets.php

```diff
@@ -127,7 +127,9 @@ function gutenberg_override_translation_file( $file, $handle ) {
 	}
 
 	// Ignore scripts that are not found in the expected `build/` location.
-	$script_path = gutenberg_dir_path() . 'build/' . substr( $handle, 3 ) . '/index.js';
+	$indexFilename = ( defined( 'SCRIPT_DEBUG') && SCRIPT_DEBUG ) ? 'index.js' : 'index.min.js';
```

Comment:
style nit: `an_underscore_convention` is used for identifiers in this PHP script. And I would make the conditional part of the name more minimal, like this:
```
$suffix = $condition ? '.js' : '.min.js'
```

Reply:
Ah yeah! I'm aware of that, but my muscle habit beat me to it this time.